plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/BasicActionHandler.java

```diff
@@ -35,7 +35,7 @@
    @Override
    @SuppressWarnings("unchecked")
    protected Class<T> deriveActionType() {
-      return (Class<T>) GenericsUtil.getGenericTypeParameterClass(getClass(), BasicActionHandler.class);
+      return (Class<T>) GenericsUtil.getActualTypeArgument(getClass(), Action.class);
```

Comment:
This override seems unnecessary since it does exactly the same as the `derviceActionType` method of the parent class (`AbstractActionHandler`)

Reply:
Done.